### ISSUE TITLE
websocket receive package size > 1023 error.

### ISSUE THREAD

**Issue by zmox** [cocos2d/cocos2d-x#15767]
win32  platform,  if server send msg size > 1023 
WebSocket::onClientReceivedData 

```
    size_t remainingSize = lws_remaining_packet_payload(_wsInstance);
    int isFinalFragment = lws_is_final_fragment(_wsInstance);
    LOGD("remainingSize: %d, isFinalFragment: %d\n", (int)remainingSize, isFinalFragment);
```

the log is :
remainingSize: 0, isFinalFragment: 0

and just receive one package size = 1023 

but other platform can receive more than one package

**Comment by minggo** [comment]
IPv6 is not supported on Windows, please refer to https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin/pull/219

**Comment by zmox** [comment]
@minggo 

so, how to fix this?

**Comment by minggo** [comment]
I have not idea, may be we should update websocket to latest version to have a try.

**Comment by dumganhar** [comment]
@zmox , try to disable permessage-inflate by
https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/network/WebSocket.cpp line598
// info.extensions = exts; // comment this line.

**Comment by minggo** [comment]
fixed in #16366
